src/main/resources/sql/bridge_healthdatarecord3.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS bridge_healthdatarecord3_<dateSuffix> (
+    id              CHAR(36)        ENCODE RAW          NOT NULL,
+    healthCode      CHAR(36)        ENCODE RAW          NOT NULL,
+    createdOn       BIGINT          ENCODE DELTA32k             ,
+    studyId         VARCHAR(50)     ENCODE BYTEDICT             ,
+    uploadId        CHAR(36)        ENCODE RAW          NOT NULL,
+    uploadDate      DATE            ENCODE RAW                  ,
+    schemaId        VARCHAR(250)    ENCODE LZO                  ,
+    schemaRevision  INT             ENCODE DELTA                ,
+    data            VARCHAR(10000)  ENCODE LZO                  ,
+    metaData        VARCHAR(5000)   ENCODE LZO                  ,
+    version         INT             ENCODE DELTA
+) SORTKEY (studyId, uploadDate, schemaId, healthCode, uploadId);
```

Comment:
They are sorted as opposed to e.g. hashed.  So unless we query a lot on record ID, there is no benefit in including it in the sort key list.